tcp_request.c

```diff
@@ -43,6 +43,8 @@ tcp_tune(evutil_socket_t handle)
         return;
     }
 
+    setsockopt(handle, IPPROTO_IP, IP_TOS, (void *) (int []) {
```

Comment:
Reduce the latency. ISPs can set up rules to favor latency on port 53, and favor throughput on port 443. Which is suboptimal if port 443 is actually used for DNS.